app/views/bookings/show.html.erb

```diff
@@ -7,6 +7,10 @@
 
   </div>
 
+  <div>
+    <p>Booking status: <%= @booking.status %></p>
```

Comment:
We might need to test it out because it might be an integer (0, 1, 2) so we might need to access the key OR we do a conditional statement like if @booking.status == 0 "Booking Pending" etc but we can work on that later I think!

Reply:
![Screenshot 2022-01-26 at 10 11 04](https://user-images.githubusercontent.com/84076465/151086325-147eeb2b-90f7-48fb-80e5-f0a5b6532e81.png)
![Screenshot 2022-01-26 at 10 10 00](https://user-images.githubusercontent.com/84076465/151086337-56bb6acd-1bc0-4202-ae81-3f65df839fcb.png)

Reply:
I think it should be okay since ActiveRecord interprets it as a string, so it should be okay but it hasn't been tested yet because pundit blocked the page :(